lib/job_tomate/triggers/webhooks/jira.rb

```diff
@@ -54,6 +54,7 @@ def run_events(webhook)
           run_events_for_issue_deleted if issue_deleted?
           run_events_for_issue_new_comment if issue_new_comment?
           run_events_for_issue_changelog if issue_changelog?
+          run_events_for_bug_without_cause if bug_without_cause?
```

Comment:
In the way I thought of the architecture, `bug_without_cause` would not be an event. You would instead reuse the more generic `IssueUpdatedStatus` [event](https://github.com/jobteaser/job_tomate/blob/master/lib/job_tomate/events/jira/issue_updated_status.rb).

So most of what you put in the event would be moved to your `SlackNotifyJIRABugIssueWithoutCauseUpdated`.